reana_client/cli/workflow.py

```diff
@@ -324,62 +364,15 @@ def workflow_status(ctx, workflow, _filter, output_format,
                                                           access_token)
             verbose_headers = ['id', 'user']
             headers = ['name', 'run_number', 'created',
-                       'status', 'progress', 'command']
+                       'status', 'progress']
```

Comment:
Shall we add progress only if the status response contains progress info? To handle the case in which  [progress is not still there](https://github.com/reanahub/reana-workflow-controller/blob/master/reana_workflow_controller/rest.py#L1100)?

Reply:
Note this won't raise an error but show 0/0 as the progress fields are initialized to 0 when a row gets added in `Job` table. I also think it shouldn't be shown in this case nonetheless.